src/task13/task13.js

```diff
@@ -1,22 +1,25 @@
-class Vec {
-    constructor(x, y) {
-        this.x = x;
-        this.y = y;
-    }
-    get length() {
-        return (Math.sqrt(Math.pow(this.x, 2) + Math.pow(this.y, 2)));
-    }
-    plus(vector) {
-        return new Vec(this.x + vector.x, this.y + vector.y);
-    }
-    minus(vector) {
-        return new Vec(this.x - vector.x, this.y - vector.y);
-    }
+function Vec(x, y) {
+    this.x = x;
+    this.y = y;
+}
+
+Vec.prototype.length = function () {
```

Comment:
I'd like you to _define_ length method other way in scope of **vec** function :)
And to make it easy to read could you please call it vector?

Reply:
or victor

Reply:
I'd like you to **define** _length_ method other way in scope of Vector function